src/Foundatio/Metrics/MetricTimer.cs

```diff
@@ -1,29 +1,29 @@
 ﻿using System;
 using System.Diagnostics;
+using System.Reactive.Concurrency;
 using System.Threading.Tasks;
 using Foundatio.Extensions;
 using Foundatio.Utility;
 
 namespace Foundatio.Metrics {
     public class MetricTimer : IAsyncDisposable {
         private readonly string _name;
-        private readonly Stopwatch _stopWatch;
+        private readonly IStopwatch _stopWatch;
         private bool _disposed;
         private readonly IMetricsClient _client;
 
         public MetricTimer(string name, IMetricsClient client) {
             _name = name;
             _client = client;
-            _stopWatch = Stopwatch.StartNew();
+            _stopWatch = SystemClock.Instance.StartStopwatch();
         }
 
         public async Task DisposeAsync() {
             if (_disposed)
                 return;
 
             _disposed = true;
-            _stopWatch.Stop();
-            await _client.TimerAsync(_name, (int)_stopWatch.ElapsedMilliseconds).AnyContext();
+            await _client.TimerAsync(_name, (int)_stopWatch.Elapsed.TotalMilliseconds).AnyContext();
```

Comment:
That's not guaranteed, unless the default package doesn't use a stopwatch (https://github.com/Reactive-Extensions/Rx.NET/blob/1ad21f5a853f2b7be5f3a7e4a8d276e7d28f0d73/Rx.NET/Source/System.Reactive.Core/Reactive/Concurrency/Stopwatch.Default.cs)